components-sdk/src/locales/es.json

```
@@ -0,0 +1,63 @@
+{
+  "components.content": "Texto",
+  "components.container": "Contenedor",
+  "components.image": "Imagen",
+  "components.file": "Archivo",
+  "components.separator": "Separador",
+  "components.button": "Botón",
+  "components.button-link": "Botón de enlace",
+  "components.select-menu": "Menú de selección",
+  "components.thumbnail": "Miniatura",
+  "components.text-input": "Entrada de texto",
+  "button.add-component": "Agregar componente",
```

I would also set this one as 
```suggestion
  "button.add-component": "Añadir componente",
```

---

I understand, for some reason I thought I always used Agregar, I'll normalize it then